test/production/app-dir/metadata-img-too-large/twitter-image/index.test.ts

```diff
@@ -1,21 +1,23 @@
 import { nextTestSetup } from 'e2e-utils'
 import { generatePNG } from '../generate-image'
 
-describe('metadata-img-too-large twitter-image', () => {
-  const { next } = nextTestSetup({
+describe('app-dir - metadata-img-too-large twitter-image', () => {
+  const { next, isTurbopack } = nextTestSetup({
     files: __dirname,
     skipStart: true,
   })
 
-  const pngFile = generatePNG(6)
+  const pngFile = generatePNG(5)
```

Comment:
why the size change? i thought we want it to be above 5MB?

Reply:
The `generatePNG` logic creates buffer as exponential, it was already exceeding 5MB on the param 5. Therefore I reduced it for memory' sake. Maybe need better logic for `generatePNG` in the first hand.